packages/notebook/style/toc.css

```diff
@@ -30,7 +30,14 @@
   background-color: var(--jp-inverse-layout-color3);
 }
 
+.jp-tocItem-content[data-running='-1']::after {
+  border-radius: 50%;
+  border: var(--jp-border-width) solid var(--jp-error-color2);
+  background-color: var(--jp-error-color2);
```

Comment:
This produces a nice UX in general. While the existing implementation is not accessible for screenreaders, we can at least make this addition colourblind-accessible by using a different symbol. For example we could use
```suggestion
  content: '⚠';
  color: var(--jp-error-color2);
```

I have not tested, so it may need some further adjustments. Or we could use 🛑 sign.

Reply:
<img width="1864" alt="image" src="https://github.com/jupyterlab/jupyterlab/assets/16680640/d5395bfd-9544-482b-98be-8c85dc69391f">
I made the color one level darker (`var(--jp-error-color1)`). color2 is slightly too light.